### PR TITLE
Add `public` directory to avoid build error

### PR DESCRIPTION
**Pull request by LingkKang**
When building (or running in development mode) the app, without `public` directory under root path, an error will occur:

``` log
2024-01-12T16:19:50.561736Z ERROR  error
error from HTML pipeline

Caused by:
    0: error from asset pipeline
    1: error taking canonical path of directory "\\\\?\\E:\\PublicRepo\\CopyClip\\public"     
    2: The system cannot find the file specified. (os error 2)
```

After this pr, the issue can be resolved:

``` log
2024-01-12T16:32:47.351923Z  INFO copying directory path="\\\\?\\E:\\PublicRepo\\CopyClip\\public"
```

---

I think this is probably caused by `trunk`:

https://github.com/Alex222222222222/CopyClip/blob/d88ad40777fb1c7c8ee68e3c398aa7bf72c81a9a/index.html#L8

I tried to remove it. After some crude testing, the app seems still work well.

Probably this is a better approach.

### CODE REVIEW

**Comment by LingkKang** [comment]
Additionally, after either way of fixing, the following step on `prerelease-build` can be removed:

https://github.com/Alex222222222222/CopyClip/blob/d88ad40777fb1c7c8ee68e3c398aa7bf72c81a9a/.github/workflows/prerelease-build.yml#L126-L128

I can submit a subsequent commit on it if needed.

**Comment by Alex222222222222** [comment]
`public` dir is for resources including images and fonts or other stuff for the frontend.
Our project currently does not have such resources. However, all resources for the frontend should go into that dir, if there is any. So, adding one empty `.gitignore` should work.

**Comment by Alex222222222222** [comment]
> Additionally, after either way of fixing, the following step on `prerelease-build` can be removed:
> 
> https://github.com/Alex222222222222/CopyClip/blob/d88ad40777fb1c7c8ee68e3c398aa7bf72c81a9a/.github/workflows/prerelease-build.yml#L126-L128
> 
> I can submit a subsequent commit on it if needed.

Thanks, that would be nice.

**Comment by LingkKang** [comment]
> public dir is for resources including images and fonts or other stuff for the frontend.
> Our project currently does not have such resources. However, all resources for the frontend should go into that dir, if there is any.

That makes sence.

I've pushed new commit with updated CI file.

File: public/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*
```
Comment (Alex222222222222) [inline public/.gitignore]: As discussion above, ignore all the files under `public` dir probably is not the behaviour that we want.

Comment (LingkKang) [reply public/.gitignore]: Sorry for that, I've updated!

**Review by Alex222222222222 — APPROVED** [review]

**Pull request merged**